modules/ROOT/pages/traversal-framework/traversal_framework_java_api.adoc

```diff
@@ -91,7 +91,7 @@ The Framework also provides various built-in evaluators such as:
 * `Evaluators.toDepth(int)`: Includes paths up to the given depth, pruning everything deeper.
 * `Evaluators.fromDepth(int)`: Includes paths from the given depth, ignoring those before and never prunes anything.
 * `Evaluators.includingDepths(int, int)`: Includes only the paths of depth equal to and between the 2 given depths.
-* `Evaluators.lastRelationshipTypeIs(Evalutation, Evaluation, RelationshipType...)`: Allows the choice of which evaluation to take based on
+* `Evaluators.lastRelationshipTypeIs(Evaluation, Evaluation, RelationshipType...)`: Allows the choice of which evaluation to take based on
```

Comment:
Thanks for the new PR, Gem! However, is it correct to have Evaluation 2x here?

Reply:
Yep, they are used differently :)

```
public static <STATE> PathEvaluator<STATE> lastRelationshipTypeIs(
            final Evaluation evaluationIfMatch,
            final Evaluation evaluationIfNoMatch,
            final RelationshipType type,
            RelationshipType... orAnyOfTheseTypes)
```

Reply:
Right! Thank you